src/frontend/language_phase.nit

```diff
@@ -0,0 +1,121 @@
+# Basic support of internationalization through the generation of id-to-string tables
+module language_phase
+
+private import annotation
+private import parser_util
+import astbuilder
+import gettext
+
+redef class ToolContext
+	# Main phase of `language`
+	var language_phase: Phase = new LanguagePhase(self, null)
+end
+
+class LanguagePhase
+	super Phase
+
+	redef fun process_annotated_node(nmodule, nat) do
+		if not nat.name == "language" then return
+
+		if not nmodule isa AModuledecl then
+			toolcontext.error(nmodule.location, "Cannot use annotation language on something other than a module")
```

Comment:
I'd go for "Error: Cannot use the annotation `language` on something other than a module."

or even: "Error: The annotation `language` can only be used on module declarations."